serve/mlc_serve/engine/staging_engine_worker.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         self.cache_manager = model_module.cache_manager
         self.tokenizer = model_module.tokenizer
         self.model_artifact_config = model_module.model_artifact_config
-
+        self.max_context_length = self.model_artifact_config.max_context_length
```

Comment:
@sunggg I'm assuming that `max_context_length` is always available in the artifact. Let me know if it is not the case.

Reply:
Correct. It is safe to assume so at least for the models of our current interests.